src/FormMigration/Controllers/TransferController.php

```diff
@@ -38,6 +39,7 @@ public function __invoke(DonationForm $formV2, TransferOptions $options)
                 wp_trash_post($formV2->id);
             }
 
+            wp_update_post(['ID' => $v3FormId, 'post_status' => DonationFormStatus::PUBLISHED()->getValue()]);
```

Comment:
Instead of directly setting it as "published," we could use the current status of the v2 form. For example, a form can be private, so the published status may not be appropriate in that case.

Reply:
Good point! Resolved by https://github.com/impress-org/givewp/pull/6987/commits/e6334d68ba9534f3c30f86873115f0462c798c60.